apps/docs/src/components/document/get-started/developers-advice.tsx

```diff
@@ -1,6 +1,6 @@
 'use client';
 import { Heading } from '@govie-ds/react';
-import { Children, isValidElement, useState } from 'react';
+import React, { Children, isValidElement, useState } from 'react';
```

Comment:
Why did we have to import `React` here ? Are we using it somewhere ?

Reply:
nope, removing